src/main/java/com/atlassian/bitbucket/jenkins/internal/trigger/BitbucketWebhookMultibranchTrigger.java

```diff
@@ -65,32 +57,5 @@ public boolean isApplicable(Item item) {
             return item instanceof MultiBranchProject;
         }
 
-        public boolean addTrigger(Item item, BitbucketSCMSource scm) {
```

Comment:
Hmm, what's going on here? This code is removed, was it unused or what's going on?